app/controllers/admin/application_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Admin::ApplicationController < ApplicationController
```

Comment:
__[rubocop]__
* Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.

<!-- Comment by [SideCI](https://sideci.com) -->